### ISSUE TITLE
Condition expression for update_item cannot reference an item attribute value

### ISSUE THREAD

**Issue by stupoid** [HENNGE/aiodynamo#41]
This happened when I was using conditions expressions with `update_item`

This used to work in botoland:
`condition=Attr("files").size().lt(Attr("max_num_files"))`

Now when I do this
`condition=F("files").size().lt(F("max_num_files"))`

it looks like it can't do attribute values?
```
|   File "/usr/local/lib/python3.8/site-packages/aiodynamo/utils.py", line 127, in low_level_serialize
|     raise TypeError(f"Unsupported type {type(value)}: {value!r}")
| TypeError: Unsupported type <class 'aiodynamo.expressions.F'>: <aiodynamo.expressions.F object at 0x7fbcac2ea130>
```

**Comment by dimaqq** [comment]
🎉🧨🎉

**Comment by ojii** [comment]
Fixed and released in 20.4.2
